packages/ui/src/app/components/FMBanner.tsx

```diff
@@ -48,8 +54,12 @@ export const FMBanner = () => {
           Don't show this again
         </Checkbox>
         <ButtonsGroup>
-          <ButtonSecondary size="medium">Speak with Community Integrators</ButtonSecondary>
-          <ButtonPrimary size="medium">Learn more</ButtonPrimary>
+          <ButtonSecondary size="medium">
+            <ButtonLink href="https://discord.com/invite/nmWPBJNT5x">Speak with Community Integrators</ButtonLink>
+          </ButtonSecondary>
+          <ButtonPrimary size="medium">
+            <ButtonLink href="https://www.joystream.org/start-here/what-is-joystream/">Learn more</ButtonLink>
+          </ButtonPrimary>
```

Comment:
Just instead of adding a local styled component could you replace these by:
```tsx
<LinkButtonSecondary size="medium" href="https://discord.com/invite/nmWPBJNT5x">
  Speak with Community Integrators
</LinkButtonSecondary>
<LinkButtonPrimary size="medium" href="https://www.joystream.org/start-here/what-is-joystream/">
  Learn more
</LinkButtonPrimary>
```
And edit `@/common/components/buttons/LinkButtons` to forward `href`.

Reply:
But LinksButtons are currently wrapped `react-router-dom` Links and thus:
1. `to` property allows only internal links
2. it is required, so we can't make it optional and pass href additionally and then decide whether it's an internal or external link.

So either I'm overlooking an easier way or we should create separate components for LinkButtons with external links

Reply:
Got it, sorry when I saw `const ButtonLink = styled(Link)` I thought it was the `react-router-dom` Link... but you're right, although I think we should probably fix these `LinkButtons` at some point, we shouldn't do it in this PR.